doc/cephfs/multifs.rst

```diff
@@ -29,6 +29,19 @@ support the new file system. The deployment technology used, e.g. cephadm, will
 also configure the MDS affinity (see: :ref:`mds-join-fs`) of new MDS daemons to
 operate the new file system.
 
+In case the data and/or metadata pools for the volume/CephFS that needs to be
```

Comment:
s/In case/If/

, the names of these pools can

Reply:
Fixed.